tests/gtests/sw_testhelpers.h

```diff
@@ -18,14 +23,54 @@ static const double
 #undef missing
 #define missing(x)  ( EQ( fabs( (x) ), SW_MISSING ) || !std::isfinite( (x) ) )
 
-extern SW_ALL SW_All;
-extern SW_OUTPUT_POINTERS SW_OutputPtrs;
-extern LOG_INFO LogInfo;
-extern PATH_INFO PathInfo;
+
+class AllTest : public::testing::Test {
+  protected:
+
+    SW_ALL SW_All;
+    PATH_INFO PathInfo;
+    LOG_INFO LogInfo;
+    SW_OUTPUT_POINTERS SW_OutputPtrs;
+
+    void SetUp() override {
+      // `memcpy()` does not work to copy from a global to local attributes
+      // since the function does not copy dynamically allocated memory
+
+      const char * masterfile_test = "files.in"; // relative to 'dir_test'

```

Comment:
Could we get rid of `const char * masterfile_test = "files.in";`? And directly do `PathInfo.InFiles[eFirst] = Str_Dup(DFLT_FIRSTFILE, &LogInfo);`? Would there be any downsides? This would avoid the need to maintain two instances of hard-coded "files.in".